clients/drcachesim/common/trace_entry.h

```diff
@@ -424,6 +424,13 @@ typedef enum {
      * warmup part of the trace ends.
      */
     TRACE_MARKER_TYPE_FILTER_ENDPOINT,
+
+    /**
+     * Indicates the start of an "rseq" (Linux restartable sequence) region.  The marker
+     * value holds the end PC of the region (this is the PC after the committing store).
```

Comment:
Right. This way seems easier for users to process as they can plan ahead.  Also, when the rseq aborted, if we had the marker at the committing store the user would then not know where it was supposed to be as it would not be present.

Reply:
Added this as a (non-doxygen) comment here and updated #5953.